common/djangoapps/startup_configurations/validate_config.py

```diff
@@ -9,34 +9,17 @@ def validate_lms_config(settings):
     """
     validate_common_config(settings)
 
-    # validate feature based configurations
-    validate_marketing_site_config(settings)
-
 
 def validate_cms_config(settings):
     """
     Validates configurations for lms and raise ValueError if not valid
     """
     validate_common_config(settings)
 
-    # validate feature based configurations
-    validate_marketing_site_config(settings)
-
 
 def validate_common_config(settings):
     """
     Validates configurations common for all apps
     """
     if not getattr(settings, 'LMS_ROOT_URL', None):
         raise ValueError("'LMS_ROOT_URL' is not defined.")
-
-
-def validate_marketing_site_config(settings):
```

Comment:
@wajeeha-khalid this code is on master right now, and it''s being removed in this PR. You may not be using `MKTG_URLS`, but other parts of the platform are. Changes to this module should be reverted.